extensions/markdown-math/notebook/katex.ts

```diff
@@ -2,7 +2,7 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-import type * as markdownIt from 'markdown-it';
+import type MarkdownIt = require('markdown-it');
```

Comment:
`require` generally shouldn't be used. Instead it's likely we should enable `esModuleInterop` in the `tsconfig`. I think the error message will tell you the exact option that's needed